### ISSUE TITLE
[Bug]: Download File URL

### ISSUE THREAD

**Issue by Jypy** [tubearchivist/tubearchivist#566]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

Linux

## Describe the bug
Download file URL on the video page have /media/ path twice and return 404.
## Steps To Reproduce
Open any video and try to download the file

## Expected behavior
Video is downloaded in the browser

### Relevant log output

```shell
/media/ is added on:
line 88 of the tubearchivist/tubearchivist/home/templates/home/video.html
line 100 of the tubearchivist/tubearchivist/api/src/search_processor.py
```

**Comment by bbilly1** [comment]
can confirm, that slipped through the refactoring.

**Comment by OmarLaham** [comment]
I'll be happy to work on this =)
Would you please assign it to me?

**Comment by bbilly1** [comment]
please go ahead. I'd suggest to fix it in the template and to leave it as is in the search processor, because that ties in to a bunch of other things.

**Comment by OmarLaham** [comment]
Thanks for the advice. I will fork and have a look.

**Comment by mylogon341** [comment]
This would be nice to have fixed - I assume this is [this](https://github.com/tubearchivist/tubearchivist/pull/567#issue-1547808110) PR recently. Is this pending release?

**Comment by bbilly1** [comment]
fix is merged and shipped in v0.4.3, please update
